src/core/fonts.js

```diff
@@ -922,12 +923,15 @@ class Font {
     let subtype = properties.subtype;
     this.type = type;
     this.subtype = subtype;
+    this.isInvalidPDFjsFont = name.startsWith("InvalidPDFjsFont");
 
     let fallbackName = "sans-serif";
-    if (this.isMonospace) {
-      fallbackName = "monospace";
-    } else if (this.isSerifFont) {
-      fallbackName = "serif";
+    if (!this.isInvalidPDFjsFont) {
```

Comment:
Why can the `fallbackName` not take any other value than "sans-serif" when a FakeFont is being used?
If this is indeed necessary, a comment would probably not hurt :-)